packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.mdx

```diff
@@ -0,0 +1,40 @@
+import {
+    Controls,
+    DocsStory,
+    Meta,
+    Stories,
+    Title,
+    Description
+} from '@storybook/blocks';
+import * as tableColumnNumberTextStories from './table-column-number-text.stories';
+
+<Meta of={tableColumnNumberTextStories} />
+
+<Title of={tableColumnNumberTextStories} />
+<Description of={tableColumnNumberTextStories} />
+
+<DocsStory
+    of={tableColumnNumberTextStories.numberTextColumn}
+    expanded={false}
+/>
+<Controls of={tableColumnNumberTextStories.numberTextColumn} />
+
+# Usage Docs
+
+## Angular Usage
+
+### Number Format Pipes
```

Comment:
I don't think table column pages are the right long term home for documenting our formatting libraries and pipes. Architecturally, the table will be just one of multiple clients of those libraries and the pipes will be just one of multiple ways those libraries are exposed.

I'd propose a structure something like:
1. a new top level section (sibling of Components, Incubating, and Tokens) called something like "Text Formatting" or "Utilities" or ["Guidelines"](https://carbondesignsystem.com/guidelines/content/overview/) (@fredvisser, welcome your input here)
2. that section contains pages about the formatting functions in nimble-components and about the pipe APIs that wraps them. It could also contain other guidance about text like styling, typography, and writing style. Or other general guidelines like Accessibility and Internationalization.
3. the table columns docs are updated to say which formatter is used to format values and to link to the formatting docs. In the future other components that use the library (e.g. numeric text field) would be updated similarly.

Thoughts on that proposed structure?

Thoughts on the urgency of making that change? I think I'd be ok deferring it until we move the formatting code into a separate library but would be fine with it happening sooner if you are excited to make it happen. If it's deferred, could you make sure it's tracked in a tech debt issue?

Reply:
Since the time you wrote this, the direction has changed to export just a single pipe that wraps a number-text-table-column-specific `NumberTextUnitFormat` object. At this point, I think it still makes sense to document the pipe as a way to match the formatting of the number-text table column in other parts of an Angular UI. I think we can hold off on documenting general formatting utilities until we actually have a separate library, but at that point, I think your proposed structure sounds good. I created issue #1790 to track this.